### ISSUE TITLE
buildifier 1.0.0 for windows does not report proper version

### ISSUE THREAD

**Issue by limdor** [bazelbuild/buildtools#782]
The official buildifier.exe found in the release for version 1.0.0 does not report the version proper
![image](https://user-images.githubusercontent.com/3073717/74081561-a2f49b00-4a50-11ea-9c08-78a43eec7455.png)

**Comment by vladmos** [comment]
Thanks. It seems the version embedding has been broken for some time, I'll fix it starting with the next release.

**Comment by indrajeet307** [comment]
It is broken for linux also

**Comment by dpursehouse** [comment]
Broken on OSX (via homebrew) too.  I bisected it to 3527f4e

**Comment by vladmos** [comment]
2.0.0 with fixed versions has just been released.

**Comment by dpursehouse** [comment]
It doesn't seem to be fixed here.

```
buildtools $ git describe --tags
2.0.0
buildtools $ bazel build --workspace_status_command=/Users/dpursehouse/git/buildtools/status.py buildifier:buildifier
INFO: Analyzed target //buildifier:buildifier (0 packages loaded, 0 targets configured).
INFO: Found 1 target...
Target //buildifier:buildifier up-to-date:
  bazel-bin/buildifier/darwin_amd64_stripped/buildifier
INFO: Elapsed time: 0.296s, Critical Path: 0.08s
INFO: 0 processes.
INFO: Build completed successfully, 1 total action
buildtools $ ./bazel-bin/buildifier/darwin_amd64_stripped/buildifier --version
buildifier version: {STABLE_buildVersion}
buildifier scm revision: {STABLE_buildScmRevision}
```

**Comment by vladmos** [comment]
If you build the binary yourself use the bazel's `--stamp` flag, or just use `--config=release`:

    $ bazel build --config=release //buildifier

**Comment by dpursehouse** [comment]
Thanks @vladmos , that fixed it!

**Comment by chenrui333** [comment]
Thanks @vladmos @dpursehouse, I have just merged and shipped the buildifier 2.0.0 in homebrew side, please try it out.
